tests/utilities/test_app.py

```diff
@@ -102,15 +119,42 @@ async def boot_and_shutdown(
         """Just a commodity shortcut for `async with app.in_running_state(): pass`, for simple cases"""
         async with self.in_running_state(
             waiting_duration_after_initialisation=waiting_duration_after_initialisation,
-            waiting_duration_post_yield=waiting_duration_before_shutdown,
+            waiting_duration_after_yield=waiting_duration_before_shutdown,
         ):
             pass
 
+    async def force_full_screen_update(
+        self, *, repaint: bool = True, layout: bool = True
+    ) -> None:
+        try:
+            screen = self.screen
+        except IndexError:
+            return  # the app may not have a screen yet
+
+        # We artificially tell the Compositor that the whole area should be refreshed
+        screen._compositor._dirty_regions = {
```

Comment:
I think I might expose this on the Screen object to avoid digging in to the internals.